sources/packages/backend/libs/services/src/notifications/notification/notification.service.ts

```diff
@@ -55,17 +56,26 @@ export class NotificationService extends RecordDataModelService<Notification> {
     notifications: SaveNotificationModel[],
     auditUserId: number,
     options?: {
-      entityManager: EntityManager;
+      disbursementId?: number;
```

Comment:
This method is generic to save any notification. It should receive the `metadata` object to be saved.